processing/src/main/java/io/druid/query/extraction/ExtractionFn.java

```diff
@@ -29,7 +29,8 @@
     @JsonSubTypes.Type(name = "partial", value = MatchingDimExtractionFn.class),
     @JsonSubTypes.Type(name = "searchQuery", value = SearchQuerySpecDimExtractionFn.class),
     @JsonSubTypes.Type(name = "javascript", value = JavascriptExtractionFn.class),
-    @JsonSubTypes.Type(name = "timeFormat", value = TimeFormatExtractionFn.class)
+    @JsonSubTypes.Type(name = "timeFormat", value = TimeFormatExtractionFn.class),
+    @JsonSubTypes.Type(name = "explicitLookup", value = ExplicitDimRenameFn.class)
```

Comment:
@drcrallen @cheddar how would you feel if we create a generic "lookup" type, and then subtypes for "map" – which I think is more explicit than `explicit` :) – and leave the option for other types of lookup such as `namespace`.

I personally feel that would make for a nicer looking api, what's your thought?